mobify/progressive-web-scaffold
# WEB-1246: Batch compress local image assets

**JIRA**: https://mobify.atlassian.net/browse/WEB-1246

## After Compression
saves 148kb,  ~36% savings
![screen shot 2017-03-08 at 3 32 34 pm](https://cloud.githubusercontent.com/assets/4543395/23729519/2b5b40a8-0417-11e7-93be-078833f29c37.png)


## Changes
- Add `imagemin-pngquant` plugin to improve PNG assets compression
- Update `clean:images` script to include files stored in the root directory of `static/img`

## How to test-drive this PR
- Add some jpg/png images into the img directory
- Run `npm run clean:images` and see compression happen
